sources/packages/api/src/route-controllers/institution-locations/institution-location.aest.controller.ts

```diff
@@ -36,4 +43,38 @@ export class InstitutionLocationAESTController extends BaseController {
       institutionId,
     );
   }
+
+  /**
+   * Controller method to retrieve institution location by id.
+   * @param institutionId
+   * @param locationId
+   * @returns institution location.
+   */
+  @Get(":institutionId/:locationId")
```

Comment:
shouldn' the endpoint be like this @Get(":locationId/institution/:institutionId/") [institution/location/:locationId/institution/:institutionId]